bionic/oneline.py

```diff
@@ -0,0 +1,19 @@
+"""
+Contains the oneline method, which is very useful for formatting error messages.
+
+This is in its own module, rather than somewhere in ``util``, because it's used in
```

Comment:
I thought about this and I agree that it would be nice to simplify the dependency story. However, it's not obvious to me that "`util` should not depend on `deps`". It seems possible to have other utilities that happen to depend on external packages. If we do more with Dask we'll probably have Dask-specific utils, etc. We could put those in their own top-level groups, but I wonder if we'll just end up having `util` and then `util_with_deps`.

On the other hand, we can always deal with that problem when it actually arises. WDYT?

Reply:
I'm fine with dealing with it later.